.github/workflows/request_workflow_review.yml

```diff
@@ -0,0 +1,25 @@
+name: WF Changes Approval
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/**
+
+jobs:
+  request_approval:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+      - name: Request Reviewers
+        run: |
+          curl -L \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/requested_reviewers \
+          -d '{"reviewers":["eriknordmark", "deitch", "milan-zededa"]}'
```

Comment:
Do we have a default embedded in other workflows? Seems like a maintenance hazard to have that.